includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,21 +1,185 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	private $wikiId;
+	private $lang;
+
+	private $data = [
+		'brand-logo' => [
+			'type' => 'brand-logo',
+			'href' => '#',
+			'elements' => [
+				[
+					'type' => 'line-image',
+					'image' => 'company/logo-fandom',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-navigation-logo-fandom',
+					]
+				],
+				[
+					'type' => 'line-image',
+					'image' => 'company/logo-wikia',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-navigation-logo-wikia',
+					]
+				],
+				[
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-header-title',
+				],
+			],
+		],
+		'brand-links' => [
+			[
+				'type' => 'link-branded',
+				'brand' => 'tv',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-brandlink-vertical-tv'
+				],
+				'href' => 'http:\/\/tv.wikia.com'
+			],
+			[
+				'type' => 'link-branded',
+				'brand' => 'games',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-brandlink-vertical-games'
+				],
+				'href' => 'http:\/\/games.wikia.com'
+			],
+			[
+				'type' => 'link-branded',
+				'brand' => 'movies',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-brandlink-vertical-movies'
+				],
+				'href' => 'http:\/\/movies.wikia.com'
+			],
+			[
+				'type' => 'links-branded',
+				'brand' => 'wikis',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-header-title',
+				],
+				'links' => [ // not branded
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)